api/mod/src/settings.py

```diff
@@ -23,6 +23,9 @@ class Settings(BaseSettings):
         f"sqlite:///{Path.home().joinpath('.annoto').joinpath('db.sqlite3')}"
     )
     debug_routes: bool = True
+    jwt_algorithm: str = "HS256"
```

Comment:
maybe we could provide some information about the HS256 in case people wants to know more about the security of the backend and session handling

Reply:
Actually, I don't know anything more about it either... So if you have something to add, feel free to just push it :)